package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "plugin-template",
+  "name": "@ubiquity-os/autoedit",
```

Comment:
```suggestion
  "name": "@ubiquity-os-marketplace/command-config",
```

Reply:
Change the project name reference everywhere appropriate.

Reply:
Updated the proiect name